include/message_struct.h

```diff
@@ -0,0 +1,12 @@
+#ifndef _MESSAGE_DATA_STATIC_H_
+#define _MESSAGE_DATA_STATIC_H_
+#include "PR/ultratypes.h"
+
+struct MessageTableEntry{
+    /* 0x0000 */ u16 textId;
+    /* 0x0002 */ u8 typePos;
+    /* 0x0003 */ //u8 pad3;
+    /* 0x0004 */ const char* segment;
+}; // size = 0x8;
```

Comment:
We like to typedef our structs. You can also remove that pad.
```suggestion
typedef struct MessageTableEntry {
    /* 0x0000 */ u16 textId;
    /* 0x0002 */ u8 typePos;
    /* 0x0004 */ const char* segment;
} MessageTableEntry; // size = 0x8;
```

Reply:
if I typedef this struct in `message_data_static.h` and then include in `z64.h` it creates differences and I don't know how to avoid that. I'm probably missing something about new struct definition inside z64.
Could define usage inside MessageContext as void* and typecast it inside z_message files but that doesn't seem right

Reply:
Do you mean `NON_MATCHING` differences? If so that, that is probably just bss reordering and I would recommend using `global_bss_check.py` in tools. If you have any questions feel free to ask in the discord.

Reply:
it was a bss problem, fixed it in 0ec018c3